### PR TITLE
Hot feature set draft

### PR DESCRIPTION
**Pull request by meganokeefe**
updated readme to put vaquero's features at front-and-center. More detail may be needed.

### CODE REVIEW

**Comment by gitschaub** [comment]
I'm not sure if I understand points 1 & 2 on the roadmap. I'm not sure if 3 is actually on our list anymore (@stephenrlouie?)

**Comment by stephenrlouie** [comment]
lets put the favicon on every page

File: docs/current/README.md [hunk 1]
```diff
@@ -16,21 +17,49 @@
...
 
 A bare metal configuration utility that network boots machines based on user defined templates. We leverage iPXE and support cloud-config, ignition, kickstart, and untyped unattend boot scripts.
 
 The only thing you need pre-installed to run Vaquero is [Docker](https://www.docker.com/).
 
 See the [Getting Started](getting-started.html) page for details on deploying Vaquero in virtualbox.
 
+## Features
+
+(last update: 11/16/16)
```
Comment (stephenrlouie) [inline docs/current/README.md]: NIT: month / year

File: docs/current/README.md [hunk 2]
```diff
@@ -16,21 +17,49 @@
...
+
+### Completed
+1. A **powerful, customizable** configuration file for your bare metal deployment, with built-in validation tools and examples
```
Comment (stephenrlouie) [inline docs/current/README.md]: not sure what this means

File: docs/current/README.md [hunk 3]
```diff
@@ -16,21 +17,49 @@
...
+2. A streamlined **command line interface**
```
Comment (stephenrlouie) [inline docs/current/README.md]: lets talk about the cli tools we have

File: docs/current/README.md (hunk 2 above)
Comment (stephenrlouie) [reply docs/current/README.md]: maybe something about single container for simplicity that runs in multiple modes

File: docs/current/README.md [hunk 4]
```diff
@@ -16,21 +17,49 @@
...
+3. The **Source of Truth** (SoT): A textual, structured, easily updatable representation of a VCS cluster, with Github integration
+4. Support for delivering assets (kernel, ramdisk, images, cloud-config) over **http**
```
Comment (stephenrlouie) [inline docs/current/README.md]: CDN support too

File: docs/current/README.md [hunk 5]
```diff
@@ -16,21 +17,49 @@
...
+5. **Federated architecture:** the ability to manage multiple data centers with a single Vaquero instance
+6. **Lights-out management:** the ability to remotely manage hardware power state
```
Comment (stephenrlouie) [inline docs/current/README.md]: not complete

File: docs/current/README.md [hunk 6]
```diff
@@ -16,21 +17,49 @@
...
+7. Workflow **automation** to provide managed installs and updates of the cluster
+8. Support for incremental (multistep) cluster provisioning
+9. **DHCP** support for provisioning on multiple subnets
```
Comment (stephenrlouie) [inline docs/current/README.md]: DHCP relay, proxy vs server mode

**Comment by stephenrlouie** [comment]
I really like the icon. 

Want to make this feature list pretty granular, more granular than the requirements list. I also want to be considerate of the space on the page, a long laundry list might not be the best display. I'll be thinking about features tonight

**Pull request merged**